engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/EngineJobExecutionActor.scala

```diff
@@ -548,11 +548,8 @@ class EngineJobExecutionActor(replyTo: ActorRef,
       // due to timeouts). That's ok, we just ignore this message in any other situation:
       stay()
     case Event(msg, _) =>
-      log.error("Bad message from {} to EngineJobExecutionActor in state {}(with data {}): {}",
-                sender(),
-                stateName,
-                stateData,
-                msg
+      jobLogger.error(
+        s"Bad message from ${sender()} to EngineJobExecutionActor in state ${stateName}(with data ${stateData}): ${msg}"
```

Comment:
It looks like these errors do NOT correlate with stalled tasks, but still seems useful to be able to link them to the relevant jobs.